src/vllm_tgis_adapter/grpc/adapters.py

```diff
@@ -107,7 +107,10 @@ async def validate_adapters(
             )
 
             # Add to cache
-            adapter_store.adapters[adapter_id] = adapter_metadata
+            # We don't cache LoRA requests because the vllm api server
```

Comment:
There's also a collision issue with `adapter_store.next_unique_id` and the unique id that the REST server will be using for cached adapters. Both servers initialize a counter at 0 and increment it for each adapter. In practice, if we stop creating or caching LoRA adapter requests here and we never enable prompt tuning, then we'll never have a collision. But I think we should go ahead and make some changes to ensure that it doesn't happen

Reply:
so I think I was a bit mistaken about the exact behavior of the tgis adapter - i thought those requests were getting forwarded to vllm at some point rather than being executed manually here. assuming I make it forward those requests to the vllm server, how is there potential for collision? I see how some id's will be "skipped", where our counter is incremented and then that doesn't eventually result in a request being issues to vllm, but how would the same counter be reused? is it possible the vllm http server is receiving requests independently of the adapter? If that's the case, don't we have to have some kind of way to share the incrementing of the counter that's atomic?